openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -104,3 +108,26 @@ def get_invalid(cls, instance):
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
+        output_node = instance.data.get("output_node")
```

Comment:
Can we add a comment about what this repair is doing. Something like:
```python
# Add a "name" node before the the output node to automatically
# force a path attribute on the geometry. Be aware that this will
# create a _single name_ and thus will merge all geometry in one shape.
```